packages/credential-governance/src/domain-hierarchy.ts

```diff
@@ -159,15 +150,14 @@ export class DomainHierarchy {
    * Retrieves list of subdomains from on-chain for a given parent domain
    * based on the ENS Registry contract logs.
    * For multi-level queries with many domains, querying the registry is slower than
-   * using the resolver contract because of the repeated RPC call.
+   * using the resolver contract because each RPC call returns only direct subdomains
    */
   public getSubdomainsUsingRegistry = async ({
     domain,
   }: {
     domain: string;
   }): Promise<string[]> => {
-    if (!domain) throw new Error('You need to pass a domain name');
-    const notRelevantDomainEndings = ['roles', 'apps'];
+    if (!domain) throw new Error('You need to pass a domain name'); // ?
```

Comment:
```suggestion
    if (!domain) throw new Error('You need to pass a domain name'); 
```
Is this intentional?

Reply:
Sorry, I forgot to remove this